### ISSUE TITLE
Could not resolve annotation @GherkinTestSuite() in `void main()`

### ISSUE THREAD

**Issue by yeongsheng-tan** [jonsamwell/flutter_gherkin#193]
pubspec.yaml file added to dev_dependencies:
```
dependencies:
  flutter:
    sdk: flutter
  cupertino_icons: ^1.0.4
  http: ^0.13.4

dev_dependencies:
  integration_test:
    sdk: flutter
  flutter_test:
    sdk: flutter
  build_runner: ^2.1.7
  flutter_gherkin: ^3.0.0-rc.9
  flutter_lints: ^1.0.4
```

Added build.yaml to root of flutter 2.8.1 (dart 2.15.1) project:
```
targets:
  $default:
    sources:
      - lib/**
      - pubspec.*
      - $package$
      # Allows the code generator to target files outside of the lib folder
      - integration_test/**.dart
```

Created `test_driver\integration_test_driver.dart` with contents:
```
import 'package:integration_test/integration_test_driver.dart' as integration_test_driver;

Future<void> main() {
  // The Gherkin report data send back to this runner by the app after
  // the tests have run will be saved to this directory
  integration_test_driver.testOutputsDirectory = 'integration_test/gherkin/reports';

  return integration_test_driver.integrationDriver(
    timeout: Duration(minutes: 90),
  );
}
```

Created `integration_test\features\counter.feature` with contents:
```
Feature: Counter

Scenario: User can increment the counter
  Given I expect the "counter" to be "0"
  When I tap the "increment" button
  Then I expect the "counter" to be "1"
```

Added `integration_test\gherkin_suite_test.dart` with contents:
```
import 'package:flutter_gherkin/flutter_gherkin_integration_test.dart'; // notice new import name
import 'package:flutter_test/flutter_test.dart';
import 'package:gherkin/gherkin.dart';

// The application under test.
import 'package:example_with_integration_test/main.dart' as app;

part 'gherkin_suite_test.g.dart';

@GherkinTestSuite()
void main() {
  executeTestSuite(
    FlutterTestConfiguration.DEFAULT([])
      ..reporters = [
        StdoutReporter(MessageLevel.error)
          ..setWriteLineFn(print)
          ..setWriteFn(print),
        ProgressReporter()
          ..setWriteLineFn(print)
          ..setWriteFn(print),
        TestRunSummaryReporter()
          ..setWriteLineFn(print)
          ..setWriteFn(print),
        JsonReporter(
          writeReport: (_, __) => Future<void>.value(),
        ),
      ],
    (World world) => app.main(),
  );
}
```

When run `flutter pub run build_runner build`, got error:
```
❯ flutter pub run build_runner build
[INFO] Generating build script...
[INFO] Generating build script completed, took 511ms

[INFO] Initializing inputs
[INFO] Reading cached asset graph...
[INFO] Reading cached asset graph completed, took 77ms

[INFO] Checking for updates since last build...
[INFO] Checking for updates since last build completed, took 772ms

[INFO] Running build...
[INFO] Running build completed, took 18ms

[INFO] Caching finalized dependency graph...
[INFO] Caching finalized dependency graph completed, took 52ms

[SEVERE] flutter_gherkin:gherkin_test_suite on integration_test/gherkin_suite_test.dart (cached):

line 1, column 337 of asset:doughnut_mobile/integration_test/gherkin_suite_test.dart: Could not resolve annotation for `void main()`.
  ╷
1 │ @GherkinTestSuite()
  │ ^^^^^^^^^^^^^^^^^^^
  ╵
[SEVERE] Failed after 96ms
pub finished with exit code 1
```

No partial files generated.

**Comment by op-perilous** [comment]
I just encountered this also, and solved it for myself by changing `gherkin_test_suite.dart` from:
```
import 'package:flutter_gherkin/flutter_gherkin_integration_test.dart'; // notice new import name
```
to:
```
import 'package:flutter_gherkin/flutter_gherkin.dart';
```

**Comment by fodil-a** [comment]
Confirmed, we should update the documentation

**Comment by jonsamwell** [comment]
@fodil-a please feel free to send through a PR :-)

**Comment by AFASbart** [comment]
This issue has been resolved with #215.
